cirq/ops/common_gates.py

```diff
@@ -982,3 +982,82 @@ def Ry(rads: float) -> RotYGate:
 def Rz(rads: float) -> RotZGate:
     """Returns a gate with the matrix e^{-i Z rads / 2}."""
     return RotZGate(rads=rads, global_shift_in_half_turns=-0.5)
+
+
+class XXGate(eigen_gate.EigenGate,
+                gate_features.TwoQubitGate,
+                gate_features.InterchangeableQubitsGate):
+    """XX interaction.
+
+    The gate implements the following unitary:
+    exp(-i t X⊗X) = [cos(t) 0 0 -isin(t)]
```

Comment:
The matrix of this gate should be `(X⊗X)^t = (-i)^t exp(-i pi t X⊗X)`. The current specified matrix is off by a global phase, and has a different scaling of `t`.